src/display/legendsWindow.cpp

```diff
@@ -16,36 +16,58 @@ void LegendsWindow::draw(int windowX, int windowY) {
     static auto pollenColor = colorToImVec4(*envColours.pollenColour);
     static auto hiveColor = colorToImVec4(*envColours.hiveColour);
     static auto locationColour = colorToImVec4(*envColours.locationColour);
+
     // radio button overlay toggle values
     static int pollenOption = 1;
     static int nectarOption = 2;
 
+    static int simStatus = Status::Play;
+
     // display window
-    ImGui::Begin("Display Options");
-    // 1. Plant options
+    ImGui::Begin("Simulation Options");
+
+    // play/pause & stop
+    ImGui::RadioButton("Play", &simStatus, Status::Play); ImGui::SameLine();
+    ImGui::RadioButton("Pause", &simStatus, Status::Pause); ImGui::SameLine();
+    ImGui::RadioButton("Stop", &simStatus, Status::Stop);
```

Comment:
Might be neater to chuck the simulation controls in a new window (although it isn't that important)

Reply:
Yeah I was thinking so too but then we'd have a third window to move around which I though might be less user-friendly